### PR TITLE
VNET extension

### PR DESCRIPTION
**Pull request by rahulgouthamDOTgs**
Adding VNET extension for public repo , Pending changes :  update index.json to add the extension and uploading .whl file to blob repo will happen later .

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `./scripts/ci/test_static.sh` locally? (`pip install pylint flake8` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli-extensions/blob/master/docs/extension_summary_guidelines.md).

### CODE REVIEW

**Comment by rahulgouthamDOTgs** [comment]
@rohit-joy  @QingqingYuan  Please review .

File: src/rdbms_vnet/setup.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from codecs import open
+from setuptools import setup, find_packages
+
+VERSION = "10.0.0"
```
Comment (williexu) [inline src/rdbms_vnet/setup.py]: Why are you starting with such a high version?

File: src/rdbms_vnet/setup.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    'Programming Language :: Python :: 3.6',
+    'License :: OSI Approved :: MIT License',
+]
+
+DEPENDENCIES = []
+
+setup(
+    name='rdbms-vnet',
+    version=VERSION,
+    description='An Azure CLI Extension to manage VNET ACL rules for Azure MySQL and Azure PostgreSQL resources',
```
Comment (williexu) [inline src/rdbms_vnet/setup.py]: Please change the description to follow our [summary_guidelines](https://github.com/Azure/azure-cli-extensions/blob/master/docs/extension_summary_guidelines.md)
ex. don't mention the CLI/extension/command-line

File: src/rdbms_vnet/setup.py (hunk 1 above)
Comment (rohit-joy) [reply src/rdbms_vnet/setup.py]: It's a remnant from private extension. Is it a blocker?

Comment (williexu) [reply src/rdbms_vnet/setup.py]: Most extensions would start with something like `1.0.0` or `0.1.0` for preview.
It's strange starting off with `10.0.0` when there weren't any lower versions.
Would recommend changing but I'll leave the decision to you guys.

**Comment by rahulgouthamDOTgs** [comment]
@rohit-joy  @williexu  @derekbekoe   test_static.sh  points to ./src/rdbms/azext_rdbms/ for pylint , do we need to make it generic ?

**Comment by williexu** [comment]
@rahulgouthamDOTgs I recently made the change: https://github.com/Azure/azure-cli-extensions/blob/master/scripts/ci/test_static.sh#L12
If you rebase, you should be able to get the updated code

**Comment by rahulgouthamDOTgs** [comment]
@rohit-joy  @williexu  it looks like pylint is referring to rdbms/azext_rdbms/_client_factory.py and hence it complains , i tried changing the name to _client_vnet_factory.py , but its still looking at rdbms/azext_rdbms/ folder ,  how do i fix this ? :(

**Comment by rohit-joy** [comment]
@rahulgouthamDOTgs, could you try renaming the rdbms_vnet/azext_rdbms folder to rdbms_vnet/azext_rdbms_vnet?

**Comment by rohit-joy** [comment]
@rahulgouthamDOTgs, looks like the static analysis issues are resolved.
Add your github user to ./.github/CODEOWNERS!

**Comment by rohit-joy** [comment]
@rahulgouthamDOTgs, is it OK to merge?

**Comment by rahulgouthamDOTgs** [comment]
@rohit-joy  yes , we are good to merge :)

**Comment by rohit-joy** [comment]
@rahulgouthamDOTgs, please fix CI issue in the index.json file.

**Pull request merged**